pkg/upgrade/jiva.go

```diff
@@ -250,7 +254,7 @@ func getContainerArguments(cfg *jivaUpdateConfig) []string {
 }
 
 func checkIfJobIsAlreadyRunning(k *client.K8sClient, cfg *jivaUpdateConfig) (bool, error) {
-	jobs, err := k.GetBatchJobs()
+	jobs, err := k.GetBatchJobs("", "cas-type=jiva,name=jiva-upgrade")
```

Comment:
Applied: https://github.com/openebs/openebsctl/pull/130#discussion_r735282615

Considering the fact that we have filtered out the pods with the labels,
it comes with the problem that if users are upgrading jobs with their own yaml files or following default doc, it won't be able to track those running pods, since they won't be having any labels as such.

Reply:
Fair point, I'll confirm with the team once.